py/picca/fitter2/parser.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import print_function
```

Comment:
no need, your print is compatible with Python3

Reply:
in python2 it will print the parenthesis. With this import it will have the same behaviour in python2 and python3